### ISSUE TITLE
{Feature request} any chance you could update it to the latest angular? EOM

### ISSUE THREAD

**Issue by quantuminformation** [jhades/angular2-library-example#6]

**Comment by shaikhspear16** [comment]
Yes please. Very useful information. Would be great if demo is created on the latest version.

**Comment by BBaysinger** [comment]
I second (third) that notion. Would be helpful to see this alongside Angular-CLI as well.

**Comment by mariohmol** [comment]
Hi.. i'm trying to make this working with same version from quickstarter.. but always getting `src/app/app.component.ts(2,28): error TS2307: Cannot find module 'angular2-library-example'.
src/app/app.module.ts(3,28): error TS2307: Cannot find module 'angular2-library-example'.`

Heres is the repo, any ideas?

https://github.com/mariohmol/angular2-library-example

**Comment by quantuminformation** [comment]
@BBaysinger I now have my own library example now using angular-cli (used for tests) : https://www.npmjs.com/package/money-input
https://github.com/QuantumInformation/money-input
